Resources/public/js/views/universaldiscovery/ez-universaldiscoverybrowseview.js

```diff
@@ -22,6 +22,7 @@ YUI.add('ez-universaldiscoverybrowseview', function (Y) {
      */
     Y.eZ.UniversalDiscoveryBrowseView = Y.Base.create('universalDiscoveryBrowseView', Y.eZ.UniversalDiscoveryMethodBaseView, [], {
         initializer: function () {
+            console.warn('UniversalDiscoveryBrowseView is deprecated');
```

Comment:
this is not the *standard* message syntax for the deprecated notice (we also use `console.log`). Also, since the complete component is deprecated, this should probably be triggered directly when the module is loaded.